continuous_integration/scripts/setup-utils

```diff
@@ -0,0 +1,258 @@
+#!/usr/bin/env bash
+
+set_darwin_build_env() {
```

Comment:
All of the Darwin related functions can be removed.

Reply:
Removed.